casbin/model/model.cpp

```diff
@@ -0,0 +1,312 @@
+#include "model.h"
+#include "assertion.h"
+#include "../util/util.h"
+#include "../log/log_util.h"

```

Comment:
This file doesn't exist in git, https://github.com/ZipoChan/casbin-cpp/tree/master/casbin/log is it ignored by `.gitignore`?

Reply:
Ok,I will ignore the x64 and x86 folder and add the log_util.h

Reply:
The x64 and x86 in packages/Microsoft.googletest.v140.windesktop.msvcstl.static.rt-dyn.1.8.1.3/lib/  is generated by visual stuido to use googletest,so i didn't delete it.

Reply:
vs will use this package is because you have added gtest to `packages.config`, it's not needed, because in vs you can use the builtin gtest

![image](https://user-images.githubusercontent.com/4054836/79091471-f7cdce00-7d7f-11ea-897c-a355c68c769d.png)

I think if you set yes here, you can delete the `packages.config` in test